src/content/en/2022/css.md

```diff
@@ -6,8 +6,9 @@ description: CSS chapter of the 2022 Web Almanac covering trends, changes, and p
 authors: [rachelandrew]
 reviewers: [j9t, meyerweb, svgeesus]
 analysts: [rviscomi]
-editors: [dereknahman]
+editors: [rviscomi]
 translators: []
+rachelandrew_bio: TODO
 results: https://docs.google.com/spreadsheets/d/1OU8ahxC5oYU8VRryQs9BzHToaXcOntVlh6KUHjm15G4/
 featured_quote: TODO
 featured_stat_1: TODO
```

Comment:
@rachelandrew could you suggest a quote and three of the most interesting stats from this chapter to be featured on the home page? For example: https://almanac.httparchive.org/en/2021/?feat=css#featured-chapter